docs/pages/getting-started.mdx

```diff
@@ -56,109 +61,43 @@ cd teleport
 
+Take a look at the [Installation Guide](installation.mdx) for more options.
+
 ## Step 1b: Configure Teleport
 
-When setting up Teleport, we recommend running it with Teleport's YAML configuration file.
+Generate a configuration file for Teleport using `teleport configure`.
+
+Acme turns on automatic TLS certificates from [Letsencrypt](https://letsencrypt.org).
+Set up email to receive updates from Letsencrypt, and use a valid DNS name for a cluster name.
 
 ```bash
-# Write a basic demo config to teleport.yaml.
-$ cat > teleport.yaml <<EOF
-teleport:
-    data_dir: /var/lib/teleport
-auth_service:
-    enabled: true
-    cluster_name: "teleport"
-    listen_addr: 0.0.0.0:3025
-    tokens:
-    - proxy,node,app:f7adb7ccdf04037bcd2b52ec6010fd6f0caec94ba190b765
-ssh_service:
-    enabled: true
-    labels:
-        env: staging
-app_service:
-    enabled: true
-    debug_app: true
-proxy_service:
-    enabled: true
-    listen_addr: 0.0.0.0:3023
-    web_listen_addr: 0.0.0.0:3080
-    tunnel_listen_addr: 0.0.0.0:3024
-EOF
-
-# Write teleport.yaml to /etc/teleport.yaml (Teleport's default config location)
-$ sudo mv teleport.yaml /etc
+$ sudo teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o
+Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
 ## Step 1c: Configure Domain Name and obtain TLS certificates using Let's Encrypt
 
 Teleport requires a secure public endpoint for the Teleport UI and for end users to connect to.
-A domain name and TLS certificates are also required. We'll use Let's Encrypt to obtain a free TLS certificate.
-
-DNS Setup:<br />
-For this setup, we'll simply use an `A` or `CNAME` record pointing to the IP/FQDN of the machine with Teleport installed.
-
-TLS Setup:<br />
-If you already have TLS certificates available you can use those. If using a new domain we recommend using `certbot`, which is free and
-simple to set up. Follow [certbot instructions](https://certbot.eff.org/) for how to obtain a certificate for your distro.
-
-<Admonition type="tip" title="Using Certbot to obtain Wildcard Certs">
-Let's Encrypt provides free wildcard certificates. Below is an example command to
-use [certbot](https://certbot.eff.org/) with DNS challenge.
-Replace `foo@example.com` with your email address.
-Replace `teleport.example.com` with the domain name you want to use to access Teleport.
-
-  ```sh
-  certbot certonly \
-    --manual \
-    --preferred-challenges=dns \
-    --agree-tos \
-    --manual-public-ip-logging-ok \
-    --email foo@example.com \
-    -d "teleport.example.com, *.teleport.example.com"
-  ```
-</Admonition>
-
-**Update `teleport.yaml`**<br />
-Once you've obtained certificates from Let's Encrypt, the below commands will update Teleport's
-config file to use your newly configured domain and TLS certificates.
+To get started use an `A` or `CNAME` record pointing to the IP/FQDN of the machine with Teleport installed.
```

Comment:
Should we also add a note to recommend adding a wildcard subdomain `*.tele` for Teleport to work with Teleport Application Access?

Reply:
Good point, will add